From 25c12d94ee4583c2166662854f8fe3b896b164ba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Mon, 26 Jun 2017 20:42:18 -0400
Subject: [PATCH 2/2] Set the CMake variable PYTHON_INSTALL_DIR to bootstrap
 workspace.

This is needed for the ament environment hook shim I've made to avoid
pulling ament_cmake_python under the workspace. In the long run we'll
need to figure out a better strategy, either fixing ament_python
packages to populate the index or... let's fix ament_python packages.
---
 debian/rules.em | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 4b9444c..2854a6f 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -27,6 +27,7 @@ export @(var[0])=@(var[1])
 # This is the package that provides the workspace files so they are not
 # available to source. However this package still needs to find ament_python
 # so we explicitly set the PYTHONPATH.
+# We also set
 export PYTHONPATH=@(InstallationPrefix)/lib/python3.5/site-packages
 
 %:
@@ -37,7 +38,7 @@ override_dh_auto_configure:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_configure @(debhelper_autoconfigure_options)
+	dh_auto_configure @(debhelper_autoconfigure_options) -DPYTHON_INSTALL_DIR=lib/python3.5/site-packages
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.7.4

